README.md

```diff
@@ -27,9 +27,15 @@ For running the modelserver the default port is `8081`.
 To change the port, use the command line argument `--port=XXXX`.
 
 ## Use the modelserver API
-If the modelserver is up and running, you can access the modelserver API via `http://localhost:8081/api/model/:modeluri`.
+If the modelserver is up and running, you can access the modelserver API via `http://localhost:8081/api/*`.
 
-The following example models can be requested and are returned as a whole in JSON format:
- - `/api/model/Coffe.ecore`
- - `/api/model/SuperBrewer3000.coffee`
- - `/api/model/SuperBrewer3000.json`
\ No newline at end of file
+The following table shows the current endpoints: 
+
+|Description|HTTP method|Path|Input|Examples
+|-|:-:|-|-|-
+|Get model|__GET__|`/api/model/:modeluri`|path parameter: `modeluri`| <ul><li>`/api/model/Coffe.ecore`</li><li>`/api/model/SuperBrewer3000.coffee`</li><li>`/api/model/SuperBrewer3000.json`</li></ul>
```

Comment:
We should rename the `api/model` endpoint to `api/models`. While we're at it we should also introduce API versioning, so the final route should be `api/v1/models`